pkg/cloud/api/resource_test.go

```diff
@@ -862,6 +879,73 @@ func TestResourceImpliedVersion(t *testing.T) {
 	}
 }
 
+func TestImpliedVersionForPlaceHolderType(t *testing.T) {
```

Comment:
Inline the struct typedef (copy them) instead of creating a dependency between the two tests (TestImpliedVersion and TestImpliedVersionForPlaceholderType)

e.g. add 

```
type st struct {
		I               int
		NullFields      []string
		ForceSendFields []string
	}
	type stA struct {
		I               int
		A               int
		NullFields      []string
		ForceSendFields []string
	}
	type stB struct {
		I               int
		B               int
		NullFields      []string
		ForceSendFields []string
	}
```

to here and revert the delete.

Reply:
done